app/controllers/app_groups_controller.rb

```diff
@@ -214,6 +214,8 @@ def update_redact_labels
       end
     end
 
+    broadcast(:redact_labels_updated, @app_group.helm_infrastructure.cluster_name)
```

Comment:
let's update this to line 222 or after the redact labels already updated

Reply:
Thanks for pointing out. Updated